tests/utils/test_minio_crud_client.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import os
+import io
+import time
+
+from pathlib import Path
+
+project_root = str(Path(__file__).resolve().parent.parent.parent)
+sys.path.append(project_root)
+
+import pathlib
+import tempfile
+import shutil
+from src.utils.minio_crud import MinioCRUDClient
+import numpy as np
+import pytest
+from minio.error import S3Error
+from PIL import Image
+
+
+ENDPOINT = os.environ.get("MINIO_ENDPOINT", "192.168.49.2:32000")
+ACCESS_KEY = "3JnbRlFAACF1v5YG"
+SECRET_KEY = "lYFakKmgSpQbcKREnjPid3CXkujxPbJ8"
```

Comment:
please don't commit creds to the code, use env variables

Reply:
updated

Reply:
thanks!